101-vm-with-standardssd-disk/azuredeploy.json

```diff
@@ -0,0 +1,249 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+      "virtualMachineName": {
+          "type": "string",
+           "defaultValue":"myVM1",
+           "metadata": {
+            "description": "The name of the VM"
+          }
+      },
+      "adminUsername": {
+          "type": "string",
+          "defaultValue":"myadmin",
+          "metadata": {
+            "description": "The admin user name of the VM"
+          }
+      },
+      "adminPassword": {
+          "type": "securestring",
+          "metadata": {
+            "description": "The admin password of the VM"
+      }
+    },
+      "diskType": {
+          "type": "string",
+          "defaultValue": "StandardSSD_LRS",
+          "metadata": {
+            "description": "The Storage type of the data Disks"
+      }
+    },
+      "location": {
+          "type": "string",
+           "defaultValue":"northeurope",
+          "metadata": {
+            "description": "The Azure Region where VM is created"
+      }
+      },
+      "virtualMachineSize": {
```

Comment:
does this depend on the diskType parameter above?  (or vice versa) - if so we should add to the description

Reply:
Yes. Done